tokio-trace-subscriber/src/registry.rs

```diff
@@ -172,9 +172,9 @@ impl RegisterSpan for IncreasingCounter {
     fn new_span(&self, _new_span: &Attributes) -> Id {
         let id = self.next_id.fetch_add(1, Ordering::SeqCst);
         let id = Id::from_u64(id as u64);
-        if let Ok(spans) = self.spans.lock() {
-            // spans.insert(id.clone(), new_span);
-        }
+        // if let Ok(spans) = self.spans.lock() {
```

Comment:
I assume this is fine to leave here?

Reply:
yeah --- this whole crate needs to be rewritten; I commented this out since it was emitting a warning